easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.15.1-foss-2023b.eb

```diff
@@ -0,0 +1,210 @@
+    }),
+    ('rsa', '4.9', {
+        'checksums': ['e38464a49c6c85d7f1351b0126661487a7e0a14a50f1675ec50eb34d4f20ef21'],
+    }),
+    ('pyasn1-modules', '0.4.0', {
+        'source_tmpl': 'pyasn1_modules-%(version)s.tar.gz',
+        'checksums': ['831dbcea1b177b28c9baddf4c6d1013c24c3accd14a1873fffaa6a2e905f17b6'],
+    }),
+    ('cachetools', '5.3.3', {
+        'checksums': ['ba29e2dfa0b8b556606f097407ed1aa62080ee108ab0dc5ec9d6a723a007d105'],
+    }),
+    ('google-auth', '2.29.0', {
+        'modulename': 'google.auth',
+        'checksums': ['672dff332d073227550ffc7457868ac4218d6c500b155fe6cc17d2b13602c360'],
+    }),
+    ('google-auth-oauthlib', '1.2.0', {
+        'checksums': ['292d2d3783349f2b0734a0a0207b1e1e322ac193c2c09d8f7c613fb7cc501ea8'],
+    }),
+    ('absl-py', '2.1.0', {
+        'modulename': 'absl',
+        'checksums': ['7820790efbb316739cde8b4e19357243fc3608a152024288513dd968d7d959ff'],
+    }),
+    ('tensorboard', '2.15.2', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['a6f6443728064d962caea6d34653e220e34ef8df764cb06a8212c17e1a8f0622'],
+    }),
+    ('keras', '2.15.0', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['2dcc6d2e30cf9c951064b63c1f4c404b966c59caf09e01f3549138ec8ee0dd1f'],
+    }),
+    ('google-pasta', '0.2.0', {
+        'modulename': 'pasta',
+        'checksums': ['c9f2c8dfc8f96d0d5808299920721be30c9eec37f2389f28904f454565c8a16e'],
+    }),
+    ('astunparse', '1.6.3', {
+        'checksums': ['5ad93a8456f0d084c3456d059fd9a92cce667963232cbf763eac3bc5b7940872'],
+    }),
+    ('tblib', '3.0.0', {
+        'checksums': ['93622790a0a29e04f0346458face1e144dc4d32f493714c6c3dff82a4adb77e6'],
+    }),
+    ('astor', '0.8.1', {
+        'checksums': ['6a6effda93f4e1ce9f618779b2dd1d9d84f1e32812c23a29b3fff6fd7f63fa5e'],
+    }),
+    ('gviz-api', '1.10.0', {
+        'source_tmpl': 'gviz_api-%(version)s.tar.gz',
+        'checksums': ['846692dd8cc73224fc31b18e41589bd934e1cc05090c6576af4b4b26c2e71b90'],
+    }),
+    ('tensorboard-plugin-profile', version, {
+        'source_tmpl': 'tensorboard_plugin_profile-%(version)s.tar.gz',
+        'checksums': ['84bb33e446eb4a9c0616f669fc6a42cdd40eadd9ae1d74bf756f4f0479993273'],
+    }),
+    (name, version, {
+        'patches': [
+            'TensorFlow-2.13.0_add-missing-system-protobuf-targets.patch',
+            'TensorFlow-2.15.1_remove-libclang-dep.patch',
+            'TensorFlow-2.15.1_remove-io-gcs-filesystem-dep.patch',
+            'TensorFlow-2.15.1_add-default-shell-env.patch',
+            'TensorFlow-2.15.1_fix-flatbuffer-license.patch',
+            'TensorFlow-2.15.1_fix-pybind11-build.patch',
+            'TensorFlow-2.15.1_fix-AVX512-eigen-compilation.patch',
+            'TensorFlow-2.15.1_fix-AVX512-eigen-compilation-gcc13.patch',
+            'TensorFlow-2.15.1_upgrade-ml_dtypes-dependency-version.patch',
```

Comment:
It looks like it is missing  TensorFlow-2.15.1_disable-avx512-extensions.patch
or isn't that needed now? @Flamefire ??

Reply:
Yes and no. I remember the discussion when we added the Patch for 2.15; https://github.com/easybuilders/easybuild-easyconfigs/pull/20358#issuecomment-2165834252

> The problematic code is not called in TF 2.15.1. But as I'm convinced the code is wrong it is better to disable it with the patch

Reply:
Please add the disable-avx512-extensions.patch